mobile/mobile_45.asm

```diff
@@ -559,18 +557,18 @@ Function114412:
 	inc bc
 	inc e
 	call z, Function1144c8
-	cp $d ; CR
+	cp $d ; "\r"
```

Comment:
(I've updated issue #686 to remind us to use `"\r"` here when rgbds 0.4.0 supports it.)